Source/Evolve/WarpXEvolve.cpp

```diff
@@ -737,27 +735,33 @@ WarpX::applyMirrors(Real time){
 }
 
 // Apply current correction in Fourier space
-#ifdef WARPX_USE_PSATD
 void
 WarpX::CurrentCorrection ()
 {
+#ifdef WARPX_USE_PSATD
```

Comment:
Shouldn't this be replaced by something like `if (WarpX::maxwell_solver_id == MaxwellSolverAlgo::PSATD)` or am I missing something? Same question for the `#ifdef WARPX_USE_PSATD` below, in the implementation of `WarpX::VayDeposition`.

Reply:
That's right, for consistency.
We do only call it in such a block in `WarpXEvolve.cpp` yet, but it would make the method more robust and self-describing. Will add.